### PR TITLE
Fix #2384: revert DSE update (back to 6.8.29)

### LINKED ISSUE #2384: Revert partial DSE update for SGv1 to get back to 6.8.29 (undo of DSE changes from #2359)
Update of DSE persistence to use 6.8.31 was incomplete in #2359, and cannot be completed quite yet (there is investigation on version compatibility). In the meantime we need to make sure `v1` can be built with consistent set of components, so let's make sure all references are to 6.8.29.

### PR DESCRIPTION
**Pull request by tatu-at-datastax**
**What this PR does**:

As per #2384 revert DSE persistence dependency to 6.8.29.

**Which issue(s) this PR fixes**:
Fixes #2384

**Checklist**
- [x] Changes manually tested
- [ ] Automated Tests added/updated
- [ ] Documentation added/updated
- [ ] CLA Signed: [DataStax CLA](https://cla.datastax.com/)

### CODE REVIEW

**Review by jeffreyscarpenter — APPROVED** [review]

File: ci/Makefile [hunk 1]
```diff
@@ -1,7 +1,7 @@
 HOSTNAME        := gcr.io
 PROJECT         := stargateio
 NAME            := stargate-builder
-VERSION         := v1.0.15
+VERSION         := v1.0.16
```
Comment (jeffreyscarpenter) [inline ci/Makefile:4]: Not sure we need to rev the builder, we can just use v1.0.14?

Comment (jeffreyscarpenter) [reply ci/Makefile:4]: but looks like it's already done so never mind

Comment (tatu-at-datastax) [reply ci/Makefile:4]: Well, I built new one just in case; wasn't 100% sure, since 3.11 version changed.

**Pull request merged**